tests/python/test_array.py

```diff
@@ -49,6 +49,9 @@ def cb(cpu, data, size):
             event = ct.cast(data, ct.POINTER(Data)).contents
             self.counter += 1
 
+        def lost_cb(lost):
+            self.assertGreater(lost, 0)
+
```

Comment:
How can you be sure there must be >0 lost?

Reply:
I assume when the callback is called there must be >0 lost count? Or would Kernel emit `PERF_RECORD_LOST` even for 0 lost samples?

If you think it's better I could just change this to `pass`

Reply:
Just checked the kernel side, yes when LOST_SAMPLE type is recorded, the lost count is >0. This kind of makes sense as well. Thanks!